update_entrydns.sh

```diff
@@ -1,12 +1,28 @@
-#!/bin/sh
+#!/bin/bash
 
-[ -z "${AUTH_TOKEN}" ] && { echo "=> AUTH_TOKEN cannot be empty" && exit 1; }
+([ -z "${AUTH_TOKEN}" ] && [ -z "${AUTH_TOKENS}" ]) && { echo "=> AUTH_TOKEN(S) cannot be empty" && exit 1; }
```

Comment:
Is this the part that needs bash? Use only "AUTH_TOKEN" and it's not needed any more.